core/src/main/java/org/axonframework/commandhandling/model/AbstractRepository.java

```diff
@@ -273,4 +281,17 @@ protected void postSave(A aggregate) {
     protected void postDelete(A aggregate) {
         //no op by default
     }
+
+    @Override
+    public void send(Message<?> message, ScopeDescriptor scopeDescription) throws Exception {
+        if (scopeDescription instanceof AggregateDescriptor) {
+            load(((AggregateDescriptor) scopeDescription).getIdentifier().toString()).handle(message);
```

Comment:
loading an aggregate may result in an Exception. In the case of an `AggregateNotFoundException`, we'd want to silently ignore (but log on debug level) the message. It is not uncommon for an Aggregate to be deleted before a deadline expires.

Reply:
With message, do you mean the `message` parameter to be send, or the `message` of the exception?